packages/web-components/src/components/tile/__stories__/tile-group.stories.ts

```diff
@@ -0,0 +1,133 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html } from 'lit-element';
```

Comment:
We should take the opportunity here to update lit imports to the recommended form. See the [v2 upgrade guide](https://lit.dev/docs/v2/releases/upgrade/#update-packages-and-import-paths). I had a [PR open to do this on CAEM](https://github.ibm.com/webstandards/carbon-for-aem/pull/196/files), never got around to tidying it up, but it stands as a good guide to the changes we need. Doing this change will align well with existing components in the repo.